runtime/autoload/provider/clipboard.vim

```diff
@@ -97,6 +97,12 @@ function! provider#clipboard#Executable() abort
     let s:copy['*'] = ['wl-copy', '--foreground', '--primary', '--type', 'text/plain']
     let s:paste['*'] = ['wl-paste', '--no-newline', '--primary']
     return 'wl-copy'
+  elseif !empty($WAYLAND_DISPLAY) && executable('waycopy') && executable('waypaste')
+    let s:copy['+'] = ['waycopy']
+    let s:paste['+'] = ['waypaste']
```

Comment:
Does this need `-t text/plain` (like `wl-copy` and `wl-paste`)?

Reply:
it does not need but can be added

Reply:
Please add it for consistency.

Reply:
added it!